Example/AppDelegate+CarPlay.swift

```diff
@@ -4,6 +4,7 @@ import MapboxNavigation
 import CarPlay
 import MapboxCoreNavigation
 import MapboxDirections
+import TestHelper
```

Comment:
i think we need a better code organization strategy here; we don't want to start bringing test code into the example app if we can avoid it...